web/styles.css

```
@@ -88,17 +89,18 @@ h1 {
 .control-button.active {
     background-color: #007bff;
     border-color: #007bff;
+    color: white;
 }
 
 .control-button svg {
     width: 18px;
     height: 18px;
-    fill: var(--text-color);
+    fill: currentColor;
```

Stylelint (17.6.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)